src/Microsoft.AspNet.Mvc.Core/Filters/IFormatFilter.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Microsoft.AspNet.Mvc.Core.Filters
+{
+    public interface IFormatFilter : IResourceFilter, IResultFilter
```

Comment:
consider separating this. You're implementing a format filter based on `IResourceFilter`, `IResultFilter`, but that's not really part of the contract.